x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authc/saml/SamlRealmSettings.java

```diff
@@ -62,7 +62,8 @@
             Setting.simpleString("signing.keystore.alias", Setting.Property.NodeScope);
     public static final Setting<List<String>> SIGNING_MESSAGE_TYPES = Setting.listSetting("signing.saml_messages",
             Collections.singletonList("*"), Function.identity(), Setting.Property.NodeScope);
-
+    public static final Setting<List<String>> REQUESTED_AUTHN_CONTEXT_CLASS_REF = Setting.listSetting("sp.req_authn_context_class_ref",
```

Comment:
I don't think we want the `sp.` prefix here.
My setting names haven't been entirely consistent, but the `sp.` settings are really "tell us about your Kibana instance", and the other settings like `force_authn` are separate.